### PR TITLE
Create code for creating accounts for bounty program

### PR DESCRIPTION
**Pull request by pablo-code-org**
This script creates teacher accounts for the crowd bounty program.
We received 100 accounts that need teacher accounts. We are creating those with this one off script.
The script is using the logic from `registrations_controller.rb` by creating a new user, validating it and updating its information.

## Testing story
Scenarios:
- Running with dry run and no changes
- Running without dry run on local development (Verified I can log in into those accounts, the password is encrypted and other attributes were correctly setup). 

 pablom@pablos-air oneoff % ./bulk_load_test_users --filename test_user_input.csv --dry-run
```
GetSecretValue: development/cdo/slack_bot_token
Options: {:revert=>false, :filename=>"test_user_input.csv", :dry_run=>true}
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Updating new existing user:<REDACTED>
[DRY_RUN] - Updating new existing user:<REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>
[DRY_RUN] - Creating new user: <REDACTED>

....
```
 pablom@pablos-air oneoff % ./bulk_load_test_users --filename test_user_input.csv 
```
pablom@pablos-air oneoff % ./bulk_load_test_users --filename test_user_input.csv          
GetSecretValue: development/cdo/slack_bot_token
Options: {:revert=>false, :filename=>"test_user_input.csv"}
Creating new user:<REDACTED>
Updating new existing user:<REDACTED>
Updating new existing user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>
Creating new user:<REDACTED>

```

## PR Checklist:

- [x] Tests provide adequate coverage
- [x] Privacy and Security impacts have been assessed
- [x] Code is well-commented
- [x] New features are translatable or updates will not break translations
- [x] Relevant documentation has been added or updated
- [x] User impact is well-understood and desirable
- [x] Pull Request is labeled appropriately
- [x] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/oneoff/bulk_load_test_users [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+options.freeze
+
+$dry_run = options[:dry_run]
+
+def get_existing_user(params)
+  user = User.find_by(email: params[:email])
+  return user
+end
+
+def create_user(user_params)
```
Comment (pablo-code-org) [inline bin/oneoff/bulk_load_test_users]: @sureshc this is the transaction that I am trying to commit. The code runs without problem but not persisting in the db. not sure if I am missing something.

I added the "user.transaction" but no luck

**Comment by pablo-code-org** [comment]
oh shoot, I just realized I merged this by mistake. It is not being used, I had no idea I could merged without an approval!

@sureshc , I would still appreciate a look at this one and happy to address any comments

**Comment by Hamms** [comment]
We really should consider setting up some kind of automated protection against accidentally merging un-reviewed branches :upside_down_face:

**Comment by pablo-code-org** [comment]
I agree! I didn't know it was possible. This happened because I was merging branch A and clicked on merge for this one!

I was surprised when I was asked to merge A and I was thinking "Wait, I already merged something". I found about this when I came to check for feedback

**Pull request merged**